addons/web/static/src/js/views/basic/basic_model.js

```diff
@@ -3774,7 +3775,7 @@ var BasicModel = AbstractModel.extend({
         if (dataPoint.model === 'res.currency') {
             session.reloadCurrencies();
         }
-        if (dataPoint.model === 'res.company') {
+        if (dataPoint.model === 'res.company' && !localStorage.getItem('running_tour')) {
```

Comment:
I have a doubt now. Are we sure it's only for 'running js tour' and not also if a user is currently on a tour ?